parachain/primitives/core/src/outbound.rs

```diff
@@ -54,9 +54,14 @@ mod v1 {
 		pub command: Command,
 	}
 
+	/// The operating mode of Channels and Gateway contract on Ethereum.
 	#[derive(Copy, Clone, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
 	pub enum OperatingMode {
+		/// Normal operations. Allow sending and receiving messages.
 		Normal,
+		/// Reject outbound messages. This allows receiving governance messages but does now allow
+		/// sending from the Ethereum side. This can be used to close off an deprecated channel or
+		/// pause the bridge for upgrade operations.
```

Comment:
Might be worthwhile saying what happens to currently enqueued messages and newly enqueued messages while the bridge is halted.

Reply:
Addressed https://github.com/Snowfork/snowbridge/pull/1076/commits/cdf70b9620cb70bdbf66918b8cfea4fdc3d89fee